daemon/mgr/cri.go

```diff
@@ -709,9 +709,10 @@ func (c *CriManager) ExecSync(ctx context.Context, r *runtime.ExecSyncRequest) (
 	var output bytes.Buffer
 	startConfig := &apitypes.ExecStartConfig{}
 	attachConfig := &AttachConfig{
-		Stdout:    true,
-		Stderr:    true,
-		MemBuffer: &output,
+		Stdout:      true,
+		Stderr:      true,
+		MemBuffer:   &output,
+		MuxDisabled: true,
 	}
 
 	err = c.ContainerMgr.StartExec(ctx, execid, startConfig, attachConfig)
```

Comment:
can we use pipeIO to check the status? I think that the spin check is not good for the performance.

BTW, if the cmd is top, the timeout can work with the `io.Copy`.

Reply:
Yes, it's ugly actually, I'll make it more elegant in the future.

Thanks for your advice.

Reply:
Could you add an issue to record this? @fuweid 
Thank @YaoZengzeng for the test fixing.

Reply:
@allencloud @YaoZengzeng I will add the issue for this. Thanks